kuma/wiki/views/create.py

```diff
@@ -30,6 +30,15 @@ def create(request):
     # Try to head off disallowed Template:* creation, right off the bat
     if not Document.objects.allows_add_by(request.user, initial_slug):
         raise PermissionDenied
+    # TODO: Integrate this into a new exception-handling middleware
+    if not request.user.has_perm('wiki.add_document'):
+        context = {
+            'reason': 'create-page',
+            'request_page_url': 'https://bugzilla.mozilla.org/form.doc',
```

Comment:
@jwhitlock Is it better to make it configurable by importing it from ``constants`` ? So its possible to change it in future as well as reuse the url